### PR TITLE
Add likelihood of molecular diagnosis functionality

### PR DESCRIPTION
**Pull request by arvkevi**

### CODE REVIEW

File: phenopy/likelihood.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        phenotype_groups = read_phenotype_groups()
+    
+    try:
+        phenotype_groups[list(phenotype_groups)[0]][f"k{k_phenotype_groups}"]
+    except KeyError:
+        logger.critical("The value for k_phenotype_groups was not valid. Please use a valid k from the phenotype_groups dictionary.")
+        raise
+
+    encoded_phenotypes = encode_phenotypes(phenotypes, phenotype_groups, hpo_network, alt2prim, k=k_phenotype_groups)
+    model_file = os.path.join(project_data_dir, 'lgb.model.pkl')
```
Comment (vgainullin) [inline phenopy/likelihood.py:53]: not a fan of hard coding paths within scripts. This belongs in config, imo. Then use config['models']['lgb'] to get model_file

Comment (vgainullin) [reply phenopy/likelihood.py:53]: or the way you get obo file above: obo_file = config.get('hpo', 'obo_file')

File: phenopy/util.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 import pandas as pd
 
-from phenopy.config import logger
+from collections import Counter
+
+from phenopy.config import logger, project_data_dir
+
+
+# path to phenotype groups
+PHENOTYPE_GROUP_FILE = os.path.join(project_data_dir, "phenotype_groups.txt")
```
Comment (vgainullin) [inline phenopy/util.py:13]: same as above here. This belongs in config.

**Review by vgainullin — CHANGES_REQUESTED** [review]

File: phenopy/util.py (hunk 2 above)
Comment (arvkevi) [reply phenopy/util.py]: done

**Review by vgainullin — APPROVED** [review]

**Pull request merged**